README.rst

```diff
@@ -1,7 +1,7 @@
 INF200 Exersices:
 =================
 
-FIRST_NAME SURNAME:
+FIRST_NAME SURNAME: ivan cherednikov
```

Comment:
Please follow the exercise descriptions
This line should only be
`Ivan Cherednikov`